examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>Add a paragraph explaining briefly the utility functions defined below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>